src/Elastic.Apm/AgentComponents.cs

```diff
@@ -182,5 +187,24 @@ public void Dispose()
 
 			CentralConfigurationFetcher?.Dispose();
 		}
+
+		private static void PrintAgentLogPreambel(IApmLogger logger)
```

Comment:
nit: small typo I think.

```suggestion
		private static void PrintAgentLogPreamble(IApmLogger logger)
```